src/services/geetest.types.d.ts

```diff
@@ -7,5 +9,9 @@ type GeeTestRegister = {
 
 type GeeTestType = {
```

Comment:
If the captcha logic is being handled in the domain layer this declaration should be in domain (ie `CaptchaProvider`) and `GeeTestService` should implement it. This is the same pattern as with `Twillio` and `IPhoneProviderService`.